index.html

```diff
@@ -4,9 +4,48 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./styles/index.css" rel="stylesheet" />
     <title>Weather Report</title>
 </head>
+
 <body>
+    <div id="layout">
```

Comment:
Nice html layout. Everything is cleanly arranged, and it even makes sense if the style sheet can't be displayed.